src/test/storage/iterables_test.cpp

```diff
@@ -263,7 +263,24 @@ TEST_F(IterablesTest, ReferenceSegmentIteratorWithIteratorsSingleChunk) {
   auto reference_segment =
       std::make_unique<ReferenceSegment>(table, ColumnID{0u}, std::make_shared<PosList>(std::move(pos_list)));
 
-  auto iterable = ReferenceSegmentIterable<int>{*reference_segment};
+  auto iterable = ReferenceSegmentIterable<int, false>{*reference_segment};
```

Comment:
Coming from an outsider, I really do not know what I am reading here. What is a `ReferenceSegmentIterable<int, false>` why is it not a `ReferenceSegmentIterable<int, EraseReferencedSegmentType::No>`?